vllm/spec_decode/spec_decode_worker.py

```diff
@@ -32,7 +34,11 @@ def create_spec_worker(*args, **kwargs) -> "SpecDecodeWorker":
     speculative_config = kwargs.get("speculative_config")
     assert speculative_config is not None
 
-    target_worker = Worker(*args, **kwargs)
+    if (speculative_config.draft_model_config.hf_config.model_type ==
```

Comment:
Right now this is only used for `mlp_speculator`, but we may want to have a global list of models somewhere that denotes the requirement for a hidden states worker